### ISSUE TITLE
iOS no-user-mode: log in

### ISSUE THREAD

**Issue by pe0ny** [toggl/mobile#1719]
In no-user-mode users are able to log in but we are not able to sync user's data.

When user wants to log in (taps "Log in" button in the drawer), we'll ask confirmation right away (before directing user to Log in screen).

Title: All current data will be lost
Content: Are you sure? You'll lose all your current data.
First option: Cancel
Second option: Clear everything

If user taps "Clear everything" option, we'll direct user to Log in screen:

![toggl ios - log in 1x](https://cloud.githubusercontent.com/assets/5064445/16517298/5fb3fa9a-3f85-11e6-9ae3-fd060fc600e2.png)

**Comment by willsb** [comment]
Should the confirmation dialog be shown even if there are no time entries?

**Comment by amulware** [comment]
@pe0ny: Good question above. Otherwise this is implemented on `ios-no-user`. Needs reviewing.

**Comment by pe0ny** [comment]
@willsb @amulware good point. if there are no time entries, then we will not show this confirmation dialog.
